src/TestSuite/ConsoleIntegrationTestCase.php

```diff
@@ -253,7 +253,7 @@ protected function _makeRunner()
         if ($this->_useCommandRunner) {
             $applicationClassName = Configure::read('App.namespace') . '\Application';
 
-            return new CommandRunner(new $applicationClassName([CONFIG]));
+            return new CommandRunner(new $applicationClassName(CONFIG));
```

Comment:
It would be wise to split this into its own line, then you can add proper phpstan doc block annotation and then it will report the string vs array issue here IMO.

Reply:
I had done this originally, but PHPStorm didn't pick it up, so I didn't bother. Maybe phpstan will.